File: app/src/main/java/jgappsandgames/smartreminderslite/status/StatusAdapter.java
package jgappsandgames.smartreminderslite.status;

// Java
import java.util.ArrayList;


// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.tasks.Task;

/**
 * StatusAdapter
 * Created by joshua on 9/4/17.
 */
class StatusAdapter extends TaskAdapterInterface {
    // Initializer
    StatusAdapter(StatusActivity activity, ArrayList<Task> tasks) {
        super(activity, activity, tasks);
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/sort/status/StatusAdapter.kt
package jgappsandgames.smartreminderslite.sort.status

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

// Save
import jgappsandgames.smartreminderssave.tasks.Task

/**
 * StatusAdapter
 * Created by joshua on 12/14/2017.
 *
 * Adapter for The Status View Lists
 */
class StatusAdapter(activity: StatusActivity, tasks: ArrayList<Task>):
        TaskAdapterInterface(activity, activity, tasks)